migrating/ossm-migrating-from-service-mesh-2-to-3-assembly.adoc

```diff
@@ -0,0 +1,25 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="ossm-migrating-from-service-mesh-2-to-3-assembly"]
+= Migrating from Service Mesh 2 to Service Mesh 3
+include::_attributes/common-attributes.adoc[]
+:context: ossm-migrating-from-2-to-3-assembly
+
+toc::[]
+
+The content contained within this migration section applies only if:
+
+* You are an existing {SMProductName} user running {SMProduct} {SMv2Version}.
+* You want to move to {SMProduct} 3.0.
+
+[IMPORTANT]
+====
+If you are not running {SMProduct} {smv2version}, you must update before you can continue. link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-versioning_ossm-upgrade[Upgrading Service Mesh].
+====
+
+From time to time during the migration process, you may need to reference, or will be directed to {SMProduct} 2.x content. It may be beneficial to open link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x[OpenShift Service Mesh 2.x] in a new tab or window for easier reference. This may be especially helpful when moving between {SMProduct} 2 `ServiceMeshControlPlane` resource content and {SMProduct} 3 `Istio` resource content.
```

Comment:
```suggestion
During the migration process, you might need to reference, or you might be directed to {SMProduct} 2.x content. It can be beneficial to open link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x[OpenShift Service Mesh 2.x] in a new tab or window for easier reference. This can be especially helpful when you move between {SMProduct} 2 `ServiceMeshControlPlane` resource content and {SMProduct} 3 `Istio` resource content.
```
- Avoid "may" in technical content: https://www.ibm.com/docs/en/ibm-style?topic=word-usage#may